### ISSUE TITLE
LiveAxis Axis.TIME format OSError when zooming/scrolling to epoch.

### ISSUE THREAD

**Issue by Benjamin-Griffiths** [domarm-comat/pglive#39]
Hey!

I believe I have found a bug when using a LiveAxis with Axis.TIME ticks.

When zooming out, or scrolling through the plot, an OSError is raised. Maybe this is when the epoch time is reached?
```
...pglive\sources\live_axis.py", line 73, in <listcomp>
    tick_strings = [datetime.datetime.fromtimestamp(value).strftime("%H:%M:%S") for value in values]
OSError: [Errno 22] Invalid argument
```

If you run the example below, and drag the chart to the right, or zoom out, it reproduces the error.

```
from pglive.kwargs import Axis
from pglive.sources.live_plot_widget import LiveAxis, LivePlotWidget
from PyQt6.QtWidgets import QApplication

app = QApplication([])

pw = LivePlotWidget()
pw.show()
plt = pw.plotItem

# time axis
time_axis = LiveAxis("bottom", text="Datetime", **{Axis.TICK_FORMAT: Axis.TIME})
plt.setAxisItems({"bottom": time_axis})

app.exec()
```

**Comment by domarm-comat** [comment]
Thanks for pointing this out. I released new version v0.7.5 with fix for this. Now, when the date is overflown tick is converted to +inf or -inf instead of raising an exception.

**Comment by Benjamin-Griffiths** [comment]
Just tested v0.7.5 and it works great - thank you!
